client/admin/components/users/index.vue

```diff
@@ -100,7 +108,8 @@ export default {
   },
   computed: {
     headers,
-    defaultPage
+    defaultPage,
+    ...mapState('auth', ['user'])
```

Comment:
Move `...mapState('auth', ['user'])` at top of the object, so you don't accidentally override previously computed props.